app/cdap/components/ConnectionList/Components/TabLabelCanBrowse/styles.ts

```diff
@@ -31,13 +31,17 @@ export const useStyles = makeStyles({
     height: '24px',
   },
   labelStyles: {
-    maxWidth: '192px',
+    maxWidth: '153px',
     whiteSpace: 'nowrap',
     fontSize: '16px',
     overflow: 'hidden',
     textOverflow: 'ellipsis',
     pointerEvents: 'none',
   },
+  labelStylesCount: {
+    fontSize: '16px',
```

Comment:
use number directly instead of number with px

Reply:
resolved